### ISSUE TITLE
Throws unclear error message with backing library is not installed on Windows

### ISSUE THREAD

**Issue by LaCuneta** [NetLogo/Vid-Extension#31]
Per [this comment](https://github.com/NetLogo/Vid-Extension/issues/24#issuecomment-1095149145) on issue #24, there is a big, nasty, unhelpful exception thrown if you try to use Vid on a system without a properly installed Microsoft Visual C++ 2010 library.  We should try to detect if that library is installed or not and display a more helpful error message.

```
java.lang.ExceptionInInitializerError
at org.nlogo.extensions.vid.VidExtension$.$anonfun$camera$1(VidExtension.scala:13)
at org.nlogo.extensions.vid.VidExtension$.guiOrHeadless(VidExtension.scala:9)
at org.nlogo.extensions.vid.VidExtension$.camera$lzycompute(VidExtension.scala:13)
at org.nlogo.extensions.vid.VidExtension$.camera(VidExtension.scala:13)
at org.nlogo.extensions.vid.VidExtension.(VidExtension.scala:21)
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at java.lang.Class.newInstance(Class.java:442)
at org.nlogo.workspace.JarLoader.extensionClassManager(JarLoader.scala:48)
at org.nlogo.workspace.ExtensionManager.$anonfun$importExtension$8(ExtensionManager.scala:159)
```

**Comment by brandesNW** [comment]
Users may be able to resolve the problem by downloading the Microsoft Visual C++ 2010 library from Microsoft and re-installing it on the computer. 
This worked for issue [Java Error NetLogo/NetLogo#2038](https://github.com/NetLogo/NetLogo/issues/2038)
We still need to update the code to detect the problem and give a meaningful error message.

**Comment by LaCuneta** [comment]
@brandesNW The code was updated to give a meaningful message, which is why this issue was closed:  https://github.com/NetLogo/Vid-Extension/commit/296a8d43cb62116bddf5d7f7c5f8ec427856b9f9

That change was included with the Vid Extension 1.1.1 release, which is published and available on the extensions manager.
